src/lib/wayfinder.ts

```diff
@@ -65,7 +65,7 @@ async function pingUpdater(
         newData[index].health = { status: "pending" };
         onUpdate(newData);
 
-        const healthJson = await fetchResult.json();
+        // const healthJson = await fetchResult.json();
```

Comment:
Aren't we checking the health now or it is not needed now ?

Reply:
No, I commented that out while testing, will revert now